deployment/configenv/xml_jlibpt/EnvHelper.cpp

```diff
@@ -316,7 +316,9 @@ const char* EnvHelper::getXMLTagName(const char* name)
    else if (!strcmp(nameLC, "dropzone"))
       return XML_TAG_DROPZONE;
    else if (!strcmp(nameLC, "eclcc"))
-      return "EclCCProcess";
+      return "EclCCServerProcess";
```

Comment:
There is disconnect about what this process should be named. The XSD uses EclCCServer, the wizard creates it as EclCCServerProcess. I believe the XSL will accept either. Not sure if a change is necessary here, just wanted to bring it up.

Reply:
Let me double check. I thought it EclCCServerProcess so I made the change